python/GafferSceneUI/LightEditor.py

```diff
@@ -379,6 +379,9 @@ def __buttonPress( self, pathListing, event ) :
 			selection[columnIndex].addPath( str( cellPath ) )
 			pathListing.setSelection( selection )
 
+		if not any( selection[i].size() for i in range( 1, len ( selection ) ) ) :
+			return False
```

Comment:
Not sure if this is just me, but it seems odd that I should be able to show the history for some other cells, by right-clicking on a cell that doesn't support history. It leads to additional confusion when other cells _aren't_ selected, and the menu isn't there. I think it might give more intuitive behaviour if we just replace L371 with `if columnIndex <= 0` instead.

Reply:
> It leads to additional confusion when other cells aren't selected, and the menu isn't there

This case I definitely agree with, fixed in a52bdd66121d8439f0c8f1a9d2cd2d9ec34c9109